mkosi/config.py

```diff
@@ -2573,10 +2577,13 @@ def finalize_defaults(namespace: argparse.Namespace, defaults: argparse.Namespac
         if any(getattr(config, s) == getattr(cli_ns, s) for config in images):
             continue
 
-        setting = SETTINGS_LOOKUP_BY_DEST[s]
-
-        die(f"{setting.long}={getattr(cli_ns, s)} was specified on the command line but is not allowed to be configured by any images.",
-            hint="Prefix the setting with '@' in the image configuration file to allow overriding it from the command line.")
+        setting = SETTINGS_LOOKUP_BY_DEST[s].long
+        a = getattr(cli_ns, s)
+        die(
+            f"{setting}={a} was specified on the command line but is not allowed to be configured by any images.",
+            hint=
+            "Prefix the setting with '@' in the image configuration file to allow overriding it from the command line."
+        )
```

Comment:
Sorry, but that's just wrong. It just makes the code harder to read. If the tool is broken, then the solution is to not use the tool.

Reply:
Well, wrong on which level? It is syntactically valid... :) The question is not a tool per se, but this line being too long with any tool.

But let's split the difference and just silence the warning here.